src/TreeBuilder.cpp

```diff
@@ -390,6 +409,9 @@ TreeBuilder::Node TreeBuilder::process(NodeID id, Variable variable) {
           << "', kind: " << drgnKindStr(variable.type) << ")"
           << (variable.isStubbed ? " STUBBED" : "")
           << (th->knownDummyTypeList.contains(variable.type) ? " DUMMY" : "");
+  // Default dynamic size to 0 and calculate fallback exclusive size
+  setSize(node, 0, 0);
+  // Keep track of total sizes of all members
```

Comment:
```suggestion
```
This comment got left behind from the previous version

Reply:
Ah, I really need to be better at this, I keep forgetting. I'll fix these and land later today :)